daemon/algod/api/algod.oas3.yml

```diff
@@ -2409,6 +2433,102 @@
         "summary": "Get application information."
       }
     },
+    "/v2/applications/{application-id}/boxes": {
+      "get": {
+        "description": "Given an application ID, it returns the box names of that application.",
+        "operationId": "GetApplicationBoxes",
+        "parameters": [
+          {
+            "description": "An application identifier",
+            "in": "path",
+            "name": "application-id",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "description": "Truncated number of box names to display. If max is not set, or max == 0, returns all box-names.",
```

Comment:
```suggestion
            "description": "Number of box names to return. If max is not set, or max == 0, returns all box-names.",
```

Reply:
I would like to phrase it: "... return up to `MaxAPIBoxPerApplication`.", since we are internally maintaining a limit on number of box names of API return.

Reply:
nvm, I would follow your suggestion